AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/MySqlPermissionRepository.cs

```diff
@@ -61,12 +61,15 @@ public async Task<Model.Permission> UpdatePermission(UpdatePermissionRequest req
         return _permissionConverter.Convert(permission);
     }
 
-    public async Task<Model.Permission> PermissionRequestResponse(PermissionRequestResponse response)
+    public async Task<Model.Permission> PermissionRequestResponse(
+        PermissionRequestResponse response
+    )
     {
         var permission = await GetPermissionFromContext(response.UserId, response.SettingGroupId);
 
```

Comment:
What if no Permission is found for the specified user and setting group? Will it cause an error? If so, shouldn't we have error handling here.

Reply:
It would throw an exception like "sequence contains no matching elements" because it relies on the .Single() method. 

Shouldn't matter here because this method should only really be hit when someone is responding to a permission request in the UI. So the Permission entity in question will exist